reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -90,13 +91,13 @@ def iterative_refinement(self, wiener_small_number=0.01, count_norm_const=1):
             self.map_3d_init.copy(),
         )
 
-        half_map_3d_f_1 = IterativeRefinement.fft_3d(half_map_3d_r_1)
-        half_map_3d_f_2 = IterativeRefinement.fft_3d(half_map_3d_r_2)
+        half_map_3d_f_1 = primal_to_fourier_3D(half_map_3d_r_1)
```

Comment:
you will likely have to reshape so that `half_map_3d_r_1.shape == (1,n_pix,n_pix,n_pix)` because batch=1 here: https://github.com/compSPI/compSPI/blob/master/compSPI/transforms.py#L49